### DISCUSSION TITLE
Looking for VCF Files

### DISCUSSION THREAD

**Discussion by YanaShimana** [now-i-know-my-abcd/docs#62] [Where in the World is…? (Help)] [answered]
I downloaded something that was labeled as having a file type of VCF, but looks like big PLINK files. I've been googling, and I found one reference to VCF files on an informational ABCD PDF, but it was from Release 3. On the ABCD Wiki, in the release notes for genetics, one of the things it says it contains is "PLINK files containing all imputed genotype data for 11,666 unique subjects". I think that's what I downloaded. I don't see any references to VCF files. Could anyone let me know where to find these/if they are no longer available? Thanks a lot!

**Comment by laikevin128** [comment, marked as answer]
Here is more info regarding the vcf files: https://wiki.abcdstudy.org/release-notes/non-imaging/genetics.html#non-tabulated-genetic-data

Modifications since initial release: Genotyping of missing individuals from previous data releases due to sample mix up or failing quality control measures. Previously these files were shared as vcf files, they are now available as binarized PLINK files with labeled RSID numbers for variants when available.
